pkgs/by-name/ma/matomo/package.nix

```diff
@@ -7,12 +7,14 @@
   nixosTests,
   nix-update-script,
 }:
-
+let
+  fetchurl' = lib.makeOverridable fetchurl;
```

Comment:
Any reason `fetchurl` isn't overridable in the first place?

Reply:
To be clear, I don't exactly want `fetchurl' = lib.makeOverridable fetchurl` to become a widespread pattern, but seeing we removed matomo-beta in unstable, I think in this particular case we still want to make it easy to override the matomo src before that PR lands.